base/kubedriver.go

```diff
@@ -0,0 +1,66 @@
+package base
+
+import (
+	"errors"
+
+	// Import to initialize client auth plugins.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"github.com/iter8-tools/iter8/base/log"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
+)
+
+// KubeDriver embeds Kube configuration, and
+// enables interaction with a Kubernetes cluster through Kube APIs
+type KubeDriver struct {
+	// EnvSettings provides generic Kubernetes options
+	*EnvSettings
```

Comment:
Could we please embed *cli.EnvSettings (from Helm) instead of our own *EnvSettings? Specifically, we do not need our own struct for this. Like [this](https://github.com/iter8-tools/iter8/blob/802ad9fbc37dc0bdb0307b4dd169cc00e5e096a3/driver/kubedriver.go#L52).

The former provides a subset of what genericclioptions provides in terms of options, but it is more than enough (it includes the kubeconfig file; namespace; etc.).

Among other things, this should eliminate:
1. The need to have our own "EnvSettings" struct.2. 
2. We don't need the NewEnvSettings func.
3. We might be able to get rid of the `k8s.io/cli-runtime v0.23.4` (all our dependencies are very popular packages including k8s packages, but there are still high severity vulnerabilities in the security alerts; it would be nice to pay attention to the number of dependencies).
***

Also, if at all we need a struct which is in the "actual" code, we don't want that struct it in the test_helpers* file (because it is not just a test_helper anymore).

Reply:
done